### DISCUSSION TITLE
Settings for selecting nodes

### DISCUSSION THREAD

**Discussion by Mayushil** [bcakmakoglu/vue-flow#1388] [Q&A] [answered]
Hello.

I need to highlight nodes from the application side. For example, select all nodes when a hotkey is pressed. But the selected nodes via selection area look different than the selected nodes via adding selected attribute (no blue square). Is there any possibility to apply the selection square to the manually selected nodes, or not to draw blue square after selection with selection area - but without crutches like deep styles rewriting?

Thanks for your attention.

**Comment by bcakmakoglu** [comment, marked as answer]
You can force the selection box by setting `nodesSelectionActive` to `true`.

```ts
const { nodesSelectionActive, addSelectedNodes } = useVueFlow()

function selectNodes(nodes) {
	addSelectedNodes(nodes)

	nodesSelectionActive.value = true
}
```

[Here's an example.](https://codesandbox.io/p/devbox/priceless-dew-3x285k?file=%2Fsrc%2FApp.vue%3A28%2C1-30%2C36&workspaceId=b0b5eb08-9090-4c77-9dc6-14c25ecc472e)
